### PR TITLE
add upload file max size control

### PR DESCRIPTION
**Pull request by lovemyliwu**

### CODE REVIEW

File: ckeditor_uploader/views.py [hunk 1]
```diff
@@ -62,6 +62,13 @@ def post(self, request, **kwargs):
                     window.parent.CKEDITOR.tools.callFunction({0}, '', 'Invalid file type.');
                     </script>""".format(ck_func_num))
 
+        if getattr(settings, 'CKEDITOR_UPLOAD_FILE_MAX_SIZE') is not None:
```
Comment (matthiask) [inline ckeditor_uploader/views.py]: This will crash if `CKEDITOR_UPLOAD_FILE_MAX_SIZE` isn't set at all.

**Comment by swalladge** [comment]
Any updates on this PR or #259? A maximum file size setting (or at least a possibility for a hook to process the image before saving) would be good.

What has prevented these PRs from being merged or worked on further?

**Comment by matthiask** [comment]
The PR hasn't been merged because 1. my comment hasn't been addressed and 2. because I'm personally not really interested in the uploader part -- I'm only using the `ckeditor` functionality, so...

**Comment by matthiask** [comment]
Thanks for your contribution. Unfortunately, it seems that this pull request cannot be merged anymore due to conflicts. Please resubmit if you're still interested in the feature. Thanks again!

**Pull request closed without merging**